acm/acm.el

```diff
@@ -867,6 +867,7 @@ The key of candidate will change between two LSP results."
             (with-current-buffer (get-buffer-create acm-doc-buffer)
               (erase-buffer)
               (insert candidate-doc)
+              (lsp-bridge-render-markdown-content)
```

Comment:
* Should I branch by custom variable?
* Is it okay for acm to depend on functions in lsp-bridge?

Reply:
Thank you! After this, please change it as you like.